modules/library/render/src/test/java/org/geotools/renderer/lite/LabelDisplacementModeTest.java

```diff
@@ -462,12 +460,7 @@ public void testMultiLineLabelDisplacementY() throws Exception {
         String refPath =
                 "./src/test/resources/org/geotools/renderer/lite/test-data/displacementMode/testMultiLineLabelDisplacementY.png";
         // java 11 makes chars quite a bit bolder
-        int tolerance = 1300;
-        if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_11)
-                || SystemUtils.IS_OS_MAC_OSX
-                || SystemUtils.IS_OS_WINDOWS) {
-            tolerance = 2000;
-        }
+        int tolerance = 2000;
```

Comment:
What happened here, Java 8 is showing the same behavior?

Reply:
Yes. I saw ~1950 as the error.

My guess is that it isn't really a Java 11 rendering algorithm difference, but perhaps a font selection difference. So what you see depends on what font it gets (which in turn is a function of the font fallback options, and what fonts are installed). I'm optimistic that a follow-up activity can make this test more robust.

Reply:
The tests are loading up Vera fonts at the beginning, and using them in the SLDs, so it should be using those exact fonts. But I agree it should be looked at in follow up.